cassandra/src/main/java/org/apache/calcite/adapter/cassandra/CassandraFilter.java

```diff
@@ -192,6 +193,9 @@ private String translateMatch(RexNode condition) {
     private static Object literalValue(RexLiteral literal) {
       Comparable<?> value = RexLiteral.value(literal);
       switch (literal.getTypeName()) {
+      case TIMESTAMP_TZ:
```

Comment:
I think we should follow existing naming convention, if local time zone is spelled out then maybe this should be `TIMESTAMP_WITH_TIME_ZONE`

Reply:
If you make this change you can move this `case` below the one beneath to preserve alphabetical order.

Reply:
The first commit in this PR uses this type name, but in a comment on Jira @julianhyde suggested the new name so the third commit changes the name to this one. The truth is that the long name is very hard to read and is very easy to mistype.

Reply:
I understand, I am not a fan of the naming style between TIMESTAMP-related types being different though, truth be told.